CCL.Importer/Proxies/MiscReplacer.cs

```diff
@@ -27,6 +36,11 @@ public MiscReplacer()
 
             CreateMap<InteriorNonStandardLayerProxy, InteriorNonStandardLayer>().AutoCacheAndMap()
                 .AfterMap(InteriorNonStandardLayerAfter);
+            CreateMap<CabTeleportDestinationProxy, CabTeleportDestination>().AutoCacheAndMap()
+                .ForMember(m => m.hoverGlow, o => o.MapFrom(m => Mapper.GetFromCache(m.hoverGlow)));
```

Comment:
Is this a case for WithCachedMember?

Reply:
Yes, but it was the one that was giving me problems when I tried the extension.